infra/charts/management-service/values.yaml

```diff
@@ -1,11 +1,11 @@
 xpManagement:
   image:
     # -- Docker registry for XP Management Service image
-    registry: docker.io/
+    registry: ghcr.io
     # -- Docker image repository for XP Management Service
-    repository: xp-management
+    repository: gojek/turing-experiments/xp-management
     # -- Docker image tag for XP Management Service
-    tag: latest
+    tag: v0.6.1
```

Comment:
Why `v0.6.1` as opposed to `v0.6.0` ? There is real change to the `v0.6.0` release since (all the new changes are for CS).

Reply:
Yeah, noticed that, updated to `v0.6.0`.